homework/shelkovich_ilya/powerCalculator/src/test/java/com/tutrit/java/quickstart/PowerCalculatorTest.java

```diff
@@ -0,0 +1,34 @@
+package com.tutrit.java.quickstart;
+
+import static org.junit.Assert.*;
+
+import com.tutrit.java.quickstart.exceptions.NegativeNumberException;
+import com.tutrit.java.quickstart.exceptions.ZeroException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PowerCalculatorTest {
+
+    PowerCalculator powerCalculator;
```

Comment:
In lecture about "JUnit" variables of test class are listed as private.

Reply:
did it